mkdocs.yml

```diff
@@ -21,16 +26,15 @@ nav:
   - mkdocs_autorefs:
     - references.py: reference/autorefs/references.md
     - plugin.py: reference/autorefs/plugin.md
-
-- Troubleshooting: troubleshooting.md
-- Contributing: contributing.md
-- Code of Conduct: code_of_conduct.md
-- Changelog: changelog.md
-- Credits: credits.md
-- License: license.md
+- Development:
+  - Contributing: contributing.md
+  - Code of Conduct: code_of_conduct.md
+  - Coverage report: coverage.md
 
 theme:
   name: material
+  features:
+  - navigation.tabs
```

Comment:
Might as well set up https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-expansion while we're here

Reply:
Indeed, will do, thanks